data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -170,5 +170,23 @@
     "bashing": 10,
     "electric": 15,
     "flags": [ "UNARMED_WEAPON", "DURABLE_MELEE", "NONCONDUCTIVE", "LIGHT_5" ]
+  },
+  {
+    "id": "tentacle_whip",
+    "type": "GENERIC",
```

Comment:
probably should be "TOOL" since it's a weapon

Reply:
The bullwhip, which I just cut and pasted for modification, is in the melee.json and it's just a generic item. This has no tool functionality, not that it matters really. I am going to look at making it a gun, or at least using gun_data, before I fully commit this again.